tasks/main.yml

```diff
@@ -7,6 +7,8 @@
     temporary_files: []
     needs_restart: null
     alive_not_expelled_instance: null
+    cached_facts_res: null
+    cached_facts: null
```

Comment:
I think that it's not necessary to do it here. Enough to do it in the end of this file. Isn't it?